float32 traj_vel_limit
---
uint32 active_errors
uint8 procedure_result
